pages/vi/profiles/cquarte1.md

```diff
@@ -1,8 +1,8 @@
 **Carlos Quarterman**
-_Albany, GA_
+#Albany, GA#
```

Comment:
\# is supposed to be only added at front. not at both end.

Reply:
Just change resolve the issue